proposals/repository.go

```diff
@@ -53,26 +57,17 @@ func (r *Repository) FetchVoting(ctx context.Context) ([]*models.GetProposalList
 
 // FetchDeposit fetches proposals with deposit status
 func (r *Repository) FetchDeposit(ctx context.Context) ([]*models.GetProposalListResultProposals, error) {
-	return r.fetch(ctx, DepositStatus)
+	return r.fetch(ctx, DepositStatus2)
 }
 
-func (r *Repository) fetch(ctx context.Context, status string) ([]*models.GetProposalListResultProposals, error) {
-	resp, err := r.apiClient.Governance.GetV1GovProposals(
-		&governance.GetV1GovProposalsParams{
-			Status:  &status,
-			Context: ctx,
-		},
-	)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get proposals with {%s} status: %w", status, err)
-	}
-
-	err = resp.GetPayload().Validate(nil)
-	if err != nil {
-		return nil, fmt.Errorf("failed to validate proposals with {%s} status response: %w", status, err)
-	}
+// FetchVoting2 fetches proposals with voting status
+func (r *Repository) FetchVoting2(ctx context.Context) ([]*query.ProposalsOKBodyProposalsItems0, error) {
+	return r.fetch2(ctx, VotingStatus2)
+}
 
-	return resp.GetPayload().Proposals, nil
+// FetchDeposit2 fetches proposals with deposit status
+func (r *Repository) FetchDeposit2(ctx context.Context) ([]*query.ProposalsOKBodyProposalsItems0, error) {
```

Comment:
don't really like these Fetch...1 and Fetch...2 — the difference is very concealed. could you please expose the differences? ideally reflect the difference in both method names and explicitly in comments

I mean you've preferred one over another during the task, right? the reason for that could imply a good description

Reply:
or, as swelf suggested in another PR, we can just get rid of the old ones :) as far as I can see, it's only governance bot and proposal checker that uses the repo so far